code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -920,15 +920,15 @@
 /datum/reagent/mental/trisyndicotin
 	name = "Trisyndicotin"
 	id = "trisyndicotin"
-	description = "Trisyndicotin is a super-strength, expensive antipsychotic medication intended for the use in interigation. Side effects include undying hatred to NanoTrasen and disrespect for authority."
+	description = "Trisyndicotin is a super-strength, expensive antipsychotic medication intended for the use in interrogation. Side effects include hostility and disrespect for authority."
 	reagent_state = LIQUID
 	color = "#888888"
 	metabolism = 0.03
 	data = 0
-	taste_description = "freedom"
-	goodmessage = list("You distrust Nanotrasen and their people.","You feel woke.","You have urges to speak out against NanoTrasen.","You feel the need to complain about NanoTrasen on the web.","You feel like things should be better.")
-	badmessage = list() //Actual Freedom.
-	worstmessage = list() //Actual Freedom.
+	taste_description = "bitterness"
+	goodmessage = list("You feel the desire to speak your mind...","You feel awakened to the truth - your coworkers are your enemies...","You have urges to speak out against your decietful coworkers...","You feel the need to complain about how much you dislike your superiors...","You feel like things should be better...")
+	badmessage = list("What did they put in you...?","Your mind feels hazy, and the faces around you waver.") 
+	worstmessage = list("Why should you have to listen to what anyone tells you..?","You have to get your coworkers before they get you...",'KILL THEM ALL BEFORE THEY KILL YOU.') 
```

Comment:
Is turning people berserk a good idea? Hallucinated ramblings might be better, and people can choose if they want to get violent or turn into vegetables.